### DISCUSSION TITLE
Using different linux distribution

### DISCUSSION THREAD

**Discussion by mamathia** [wolveix/satisfactory-server#232] [Ideas]
Hello,
I usually use the minimum Alpine distrib for docker images.
I see you used the Abuntu one instead.
Was streamcmd Apline image tested?
Is there any chance that we can use Alpine image as a base?
ty :)

**Comment by wolveix** [comment]
Hey!

Yes, we tested Alpine before, I generally only ever use Alpine unless there's a reason not to. In this case, I recall that several system dependencies were unresolvable, or at least would be more hassle than it's realistically worth. The image is 187MB, which isn't bad given that game files alone are going to take up several GB.

PRs are always welcome if you feel like taking a stab at it, though I'd recommend against it :)
